route-controller/cmd/main.go

```diff
@@ -0,0 +1,204 @@
+	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
+		"Enable leader election for controller manager. "+
+			"Enabling this will ensure there is only one active controller manager.")
+
+	configPath, found := os.LookupEnv("CONTROLLERSCONFIG")
+	if !found {
+		panic("CONTROLLERSCONFIG must be set")
+	}
+
+	controllerConfig, err := config.LoadFromPath(configPath)
+	if err != nil {
+		errorMessage := fmt.Sprintf("Config could not be read: %v", err)
+		panic(errorMessage)
+	}
+
+	logger, atomicLevel, err := tools.NewZapLogger(controllerConfig.LogLevel)
+	if err != nil {
+		panic(fmt.Sprintf("error creating new zap logger: %v", err))
+	}
+
+	ctrl.SetLogger(logger)
+
+	log.SetOutput(&tools.LogrWriter{Logger: ctrl.Log, Message: "HTTP server error"})
+
+	ctrl.Log.Info("starting Korifi controllers", "version", version.Version)
+
+	conf := ctrl.GetConfigOrDie()
+
+	mgr, err := ctrl.NewManager(conf, ctrl.Options{
+		Scheme: scheme,
+		WebhookServer: webhook.NewServer(webhook.Options{
+			Port: 9443,
+		}),
+		Metrics: metricsserver.Options{
+			BindAddress: metricsAddr,
+		},
+		HealthProbeBindAddress: probeAddr,
+		LeaderElection:         enableLeaderElection,
+		LeaderElectionID:       "13c200ec.cloudfoundry.org",
+	})
+	if err != nil {
+		setupLog.Error(err, "unable to initialize manager")
+		os.Exit(1)
+	}
+	controllersClient := k8s.IgnoreEmptyPatches(mgr.GetClient())
+	if os.Getenv("ENABLE_CONTROLLERS") != "false" {
+		controllersLog := ctrl.Log.WithName("controllers")
+
+		if !controllerConfig.DisableRouteController {
+			if err = routeControllers.NewReconciler(
+				controllersClient,
+				mgr.GetScheme(),
+				controllersLog,
+				controllerConfig,
+			).SetupWithManager(mgr); err != nil {
+				setupLog.Error(err, "unable to create controller", "controller", "CFRoute")
+				os.Exit(1)
+			}
+		}
+	}
+	// Setup webhooks with manager
+	if os.Getenv("ENABLE_WEBHOOKS") != "false" {
+
+		(&appswebhook.AppRevWebhook{}).SetupWebhookWithManager(mgr)
+
+		uncachedClient, err := client.New(mgr.GetConfig(), client.Options{
+			Scheme: scheme,
+		})
+		if err != nil {
+			setupLog.Error(err, "unable to create uncached client")
+			os.Exit(1)
+		}
+
+		if err = routeswebhook.NewValidator(
+			validation.NewDuplicateValidator(coordination.NewNameRegistry(uncachedClient, routeswebhook.RouteEntityType)),
+			controllerConfig.CFRootNamespace,
+			uncachedClient,
+		).SetupWebhookWithManager(mgr); err != nil {
+			setupLog.Error(err, "unable to create webhook", "webhook", "CFRoute")
+			os.Exit(1)
+		}
+
+		if err = korifiv1alpha1.NewCFRouteDefaulter().SetupWebhookWithManager(mgr); err != nil {
+			setupLog.Error(err, "unable to create webhook", "webhook", "CFRoute")
+			os.Exit(1)
+		}
+
+		versionwebhook.NewVersionWebhook(version.Version).SetupWebhookWithManager(mgr)
+		controllersfinalizer.NewControllersFinalizerWebhook().SetupWebhookWithManager(mgr)
```

Comment:
`NewControllersFinalizerWebhook` is already configured in controllers. No need to have it here as well